cli/test/org/partiql/cli/CliTest.kt

```diff
@@ -112,4 +112,13 @@ class CliTest {
 
         assertEquals("<<{'a': 1}>>", actual)
     }
+
+    @Test
+    fun withPartiQLPrettyOutput() {
+        val subject = makeCli("SELECT * FROM input_data", "{a: 1, b: 2}",
+            outputFormat = OutputFormat.PARTIQL_PRETTY)
```

Comment:
nit: could move onto the same line as the previous test

```suggestion
        val subject = makeCli("SELECT * FROM input_data", "{a: 1, b: 2}", outputFormat = OutputFormat.PARTIQL_PRETTY)
```